jOOQ/src/main/java/org/jooq/InsertOnConflictDoActionStep.java

```diff
@@ -59,11 +59,17 @@
  *
  * @author Lukas Eder
  */
-public interface InsertOnConflictDoUpdateStep<R extends Record> {
+public interface InsertOnConflictDoActionStep<R extends Record> {
 
```

Comment:
We cannot rename types due to backwards compatibility constraints. It's OK if the type name doesn't fully match all of its features...